shopfloor_mobile/static/wms/src/scenario/location_content_transfer.js

```diff
@@ -132,23 +132,21 @@ export var LocationContentTransfer = Vue.component("location-content-transfer",
             const data = this.state.data;
             let res = {
                 has_records: true,
-                records: data.move_lines,
+                records: [].concat(data.move_lines || [], data.package_levels || []),
                 location_src: null,
                 location_dest: null,
                 _multi: true,
-                _type: "move_line",
+                _type: "mix",
```

Comment:
ah, so they are not mutually exclusive now?

Reply:
they weren't

Reply:
oh, right!